Library/Formula/postgresql.rb

```diff
@@ -72,13 +74,18 @@ def install
       ENV.append %w{CFLAGS LDFLAGS}, "-arch #{Hardware::CPU.arch_32_bit}"
     end
 
+    system "pkill postgres || true"
     system "./configure", *args
     system "make", "install-world"
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    if (var/"postgres").exist?
+      mv var/"postgres", var/"postgres-#{version}"
```

Comment:
Someone will do it and it might look like data loss if we hide their databases from them. :) I don't know that it should terminate abnormally but printing a "we didn't migrate your data" message is probably helpful.